lua/opencode/ui/completion/commands.lua

```diff
@@ -76,7 +76,9 @@ local command_source = {
           vim.api.nvim_win_set_cursor(0, { 1, #item.insert_text + 1 })
           return
         end
-        item.data.fn()
+        vim.defer_fn(function()
+          item.data.fn()
+        end, 10) -- slight delay to allow completion menu to close, this prevent a weird bug with mini.pick where it displays an empty window with `BlinkDonotRepeatHack` text inserted
```

Comment:
I've seen things like that before. I'm curious if vim.schedule would've been enough or if it really requires deferring for a short time?

Reply:
Vim schedule was not working unfortunately